pyat/at/tracking/patpass.py

```diff
@@ -93,6 +100,9 @@ def patpass(ring, r_in, nturns=1, refpts=None, losses=False, pool_size=None,
         coordinates at which the particle is lost. Set to zero for particles
         that survived
     """
+    if multiprocessing.get_start_method(allow_none=True) != start_method:
+        multiprocessing.set_start_method(start_method, force=True)
+
     if refpts is None:
```

Comment:
I cannot find the "force=True" keyword in the documentation… Does it work ?
You are not allowed to call set_start_method twice. What will happen if a user tries to set it a 2nd time ? If it raises an error, your previous try/catch mentioning that it failed may be safer…

Reply:
Yes it works, this thing is really not well documented...with it you can set it as many times as you and also overwrite it in case it was previously set as you saw in grappa for instance